composer.json

```diff
@@ -46,11 +46,13 @@
         }
     ],
     "require": {
+        "php": "^7.0",
         "robthree/twofactorauth": "^1.5.1"
     },
     "require-dev": {
         "hiqdev/hidev-php": "dev-master",
-        "hiqdev/hidev-hiqdev": "dev-master"
+        "hiqdev/hidev-hiqdev": "dev-master",
+        "phpunit/phpunit": "^9.2"
```

Comment:
phpunit ^9.2 requires a higher php version. I think you must go back to ^6.5 with php ^7.0

Reply:
changed locally, push later